manifests/charts/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -53,6 +53,13 @@
     # reportBatchMaxTime is the max waiting time before the telemetry data of a request is sent to the mixer server
     reportBatchMaxTime: {{ .Values.mixer.telemetry.reportBatchMaxTime }}
     {{- end }}
+
+    # The namespace to treat as the administrative root namespace for
+    # Istio configuration. When processing a leaf namespace Istio will search for
+    # declarations in that namespace first and if none are found it will
+    # search in the root namespace.
+    rootNamespace: {{ .Values.global.configNamespace }}
```

Comment:
I think we should do the opposite and change the EnvoyFilters namespace to `.values.meshConfig.rootNamespace | default .Release.Namespace`. Our whole setup with the namespace stuff is a complete mess as a legacy of some design ideas that are no longer applicable, but one thing we did do since 1.5 is replace `configNamespace` with `istioNamespace` at least so we probably don't want to add this back

Reply:
So If I understand correctly, the notion of a separate `configNamespace` is not recommended?

Reply:
The problem is there was like 8 different namespace settings and they all were sort of not correct :slightly_smiling_face: . confignamespace was used as the namespace where pilot and galley were deployed, as far as know it never represented the rootNamespace. Then it seems telemetry v2 filters are also using configRootNamespace which is not actually applying to the real `rootNamespace` field...